build-container.ps1

```diff
@@ -32,6 +32,13 @@ if($Tag -eq $null -or $Tag -eq ""){
 }
 $arglist += "build"
 
+# Read arguments from go.env file
+$lines = Get-Content -Path 'go.env'
+foreach ($line in $lines) {
+    $arglist += "--build-arg"
+    $arglist += "$line"
```

Comment:
Did you test it work as expected? I feel like we will lack of some spaces in the constructed string?

Reply:
I was also confused at first, but the thing is that `arglist` is a list, so I guess `+=` adds a new element at the end, it's not string concatenation

Note that this logic is basically copy pasted this already existing piece of code https://github.com/DataDog/datadog-agent-buildimages/blob/a4236ffb5b418a69dcc3c8c2f5c63c6a76397144/build-container.ps1#L37-L38

Reply:
I didn't test it manually but the pipeline is green, including the [pipeline](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/pipelines/25601992) triggered in `datadog-agent`

Reply:
Ah yes you're right, this is not string concatenation! Looks good then!